library/src/main/java/com/google/android/exoplayer2/upstream/ContentDataSource.java

```diff
@@ -136,22 +136,13 @@ public Uri getUri() {
   @Override
   public void close() throws ContentDataSourceException {
```

Comment:
Changes in this file appear to be irrelevant to this CL? Probably a merge error? Please revert if so?